### DISCUSSION TITLE
Failed to create symbolic link

### DISCUSSION THREAD

**Discussion by kkin1995** [iraf-community/iraf#150] [Q&A] [answered]
The exact error is:
```
Checking for /iraf symlink ...                                 
ln: failed to create symbolic link '/iraf' -> '': No such file or directory
```

I think because of this error, the next step in the installation is not possible:
```
export PATH=/path/to/iraf/bin/:$PATH
```

Inside the local folder, there is no folder called bin. Any idea how to fix this error?

I'm using Peppermint OS which is a Ubuntu derivative.

**Comment by olebole** [comment]
What exactly did you do? And which IRAF version did you use?
BTW, Ubuntu already comes with IRAF, and I would guess that the same is true for Peppermint OS. Why don't you use that one?

**Comment by kkin1995** [comment]
Hi,

I downloaded the source code from the latest [snapshot](https://github.com/iraf-community/iraf/releases/tag/v2.16.1%2B2018.11.0) . Version number 2.16.1.

And, then followed the instructions given on the [IRAF Community Website](https://iraf-community.github.io/install.html) .

Oh, I did not know that Ubuntu comes with IRAF. In that case, could you please tell me as to how to run/use the default installation?

**Comment by olebole** [comment]
You can just do a `sudo apt install iraf`, if it is there.
If you want to compile yourself, you can ignore this error; the `/iraf` directory is not used anymore. The issue is actually fixed with #85, we just didn't create a new release.

**Comment by kkin1995** [comment]
Thank You ! Just in case, I need to compile from source in the future, where do I find the IRAF bin folder to add to PATH?

**Comment by olebole** [comment, marked as answer]
This is `/usr/local/bin`, which usually should exist by default.

**Comment by kkin1995** [comment]
Right. Thanks for your help !
